.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - develop
+
+env:
+  NODE_ENV: test
+  DATABASE_URL: postgres://postgres:password@localhost:5432/approved-premises-test
+  API_CLIENT_ID: approved-premises
+  API_CLIENT_SECRET: clientsecret
+  OPENSEARCH_URL: http://localhost:9200/
+  OPENSEARCH_INDEX_NAME: bed_availability_test
```

Comment:
We'll probably be using opensearch later, but more than likely in the backend, so can get rid of these lines